vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go

```diff
@@ -139,6 +139,8 @@ var _ = DescribeSanity("Controller Service [Controller Server]", func(sc *Sanity
 				case csi.ControllerServiceCapability_RPC_PUBLISH_READONLY:
 				case csi.ControllerServiceCapability_RPC_CLONE_VOLUME:
 				case csi.ControllerServiceCapability_RPC_EXPAND_VOLUME:
+				// TODO(dyzz) Make the actual change in CSI Sanity
+				case csi.ControllerServiceCapability_RPC_LIST_VOLUMES_PUBLISHED_NODES:
```

Comment:
this will get squashed next time we update vendor. Should we wait for csi-test changes first?

Reply:
yeah, waiting on a new release - do you have an ETA?